src/app/page.module.css

```
@@ -36,13 +36,13 @@
 	font-size: 3rem;
 	margin-bottom: 0.5rem;
-	margin-top: 100px;
+	margin-top: 10px;
 }
 
 .intro h3 {
 	font-weight: bold;
 	margin-bottom: 1rem;
-	margin-top: 100px;
+	margin-top: 50px;
 	margin-bottom: 40px;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

margin-bottom is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)